src/components/graph/menus/modification-list-item.js

```diff
@@ -73,6 +73,10 @@ export const ModificationListItem = ({
             return modif.lineToAttachToId;
         } else if (modif.type === 'LINES_ATTACH_TO_SPLIT_LINES') {
             return modif.attachedLineId;
+        } else if (modif.type === 'DELETE_VOLTAGE_LEVEL_ON_LINE') {
+            return modif.attachedLineId;
+        } else if (modif.type === 'DELETE_ATTACHNING_LINE') {
```

Comment:
there is a typo here: DELETE_ATTACHNING_LINE -> DELETE_ATTACHING_LINE
as a consequence, the modification name has no computed label